### ISSUE TITLE
The disabled Pager's arrows inside the PDFViewer doesn't have aria-disabled="true"attribute

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-ui-core#5972]
### Bug report

The arrows inside the Pager component, part of the  PDFViewer's ToolBar doesn't have **aria-disabled="true"** attribute when they are disabled. The scenarios in which arrows are being disabled are when the current page in the PDFViewer is the first or the last one of the loaded PDF file. 

### Reproduction of the problem
1. Open this example
2. While the first page of the document is visible, inspect the arrows in the pager and see the available attributes

### Current behavior
The DOM elements of the disabled arrows look like the following:
`<a href="#" aria-label="Go to the first page" title="Go to the first page" class="k-button k-button-icon k-flat k-pager-nav k-pager-first k-state-disabled" data-page="1"><span class="k-icon k-i-arrow-end-left"></span></a>`

### Expected/desired behavior
When the element is disabled it should have aria-disabled="true" attribute like the following:
`<a href="#" aria-label="Go to the first page" title="Go to the first page" class="k-button k-button-icon k-flat k-pager-nav k-pager-first k-state-disabled" data-page="1" aria-disabled="true"><span class="k-icon k-i-arrow-end-left"></span></a>`

### Environment

* **Kendo UI version:** 2020.2.617
* **jQuery version:** x.y
* **Browser:** [all]

**Comment by nelito987** [comment]
When PDFViewer is used on Android device and Talkback is enabled, next and previous buttons are available to focus, but does not change the page to next/previous. The issue is reproducible on [our Demos](https://demos.telerik.com/kendo-ui/pdfviewer/index)
Reported in ticket: 1484574
